app/connectors/connectors/jms/activemq-connectors/activemq-model/src/main/java/io/syndesis/connector/jms/AbstractActiveMQConnector.java

```diff
@@ -39,6 +39,15 @@
     @Metadata(label = "basic", description = "Password for authorization credential")
     private String password;
 
+    @Metadata(label = "basic", description = "Client ID for durable subscriptions")
+    private String clientID;
+
+    @Metadata(label = "basic", description = "AMQ Broker X.509 PEM Certificate")
+    private String brokerCertificate;
```

Comment:
I don't see these added to the `deployment.json` or Camel Connector JSON schema files

Reply:
Same as the comment below.